src/sagemaker/serve/builder/model_builder.py

```diff
@@ -64,6 +64,7 @@
 )
 from sagemaker.workflow.entities import PipelineVariable
 from sagemaker.huggingface.llm_utils import get_huggingface_model_metadata
+from sagemaker_schema_inference_artifacts.huggingface import remote_schema_retriever
```

Comment:
Move this import to be on line 718


            try:
               from sagemaker_schema_inference_artifacts.huggingface import remote_schema_retriever
                sample_inputs, sample_outputs = task.retrieve_local_schemas(model_task)
            except ValueError:

Reply:
done